lib/assets/javascripts/_modules/collapsible-navigation.js

```diff
@@ -49,11 +49,11 @@
     function toggleHeading ($topLevelItem) {
       var isOpen = $topLevelItem.hasClass('is-open')
       var $heading = $topLevelItem.find('> a')
-      var $body = $topLevelItem.find('.collapsible__body')
+      var $button = $topLevelItem.find('.collapsible__toggle')
       var $toggleLabel = $topLevelItem.find('.collapsible__toggle-label')
 
       $topLevelItem.toggleClass('is-open', !isOpen)
-      $body.attr('aria-expanded', isOpen ? 'false' : 'true')
+      $button.attr('aria-expanded', isOpen ? 'false' : 'true')
       $toggleLabel.text(isOpen ? 'Expand ' + $heading.text() : 'Collapse ' + $heading.text())
```

Comment:
I appreciate this isn't code that you've written, but I found it quite confusing when reviewing and I'm wondering if there's anything we can do to make it clearer.

I thought that the `aria-expanded` attribute would be incorrect, because it's being set to `'false'` when `isOpen` is `true`, which seems backwards. However, `isOpen` is confusingly named, as it represents the state of the heading _before_ toggling, rather than the state after – which is what we care about when setting the `aria-expanded` attribute and the toggle text.

Any thoughts? Perhaps we could flip the boolean when assigning it at the top of the function rather than when calling `toggleClass`? Or rename the variable to make it clearer?

Reply:
Makes sense - refactored and updated.